### DISCUSSION TITLE
promote the mod

### DISCUSSION THREAD

**Discussion by AViewFromTheTop** [FrozenBlock/WilderWild#80] [Ideas]
@MerpisMe you should probably post a screenshot of the mod with a link to CF and Modrinth every couple days on the twitter account we need much more engagement than this
of course before CF, just modrinth

**Comment by Soro335** [comment]
Also Reddit, maybe youtubers

**Comment by mapleexists** [comment]
> @MerpisMe you should probably post a screenshot of the mod with a link to CF and Modrinth every couple days on the twitter account we need much more engagement than this
> of course before CF, just modrinth

alright, sure
but also we have a TON of engagement on the discord server, people are advertising the mod even outside of our server. the issue is just the fact that people don't know what modrinth is so they download it from the discord server instead
